include/openssl/ssl.h

```diff
@@ -1397,7 +1397,7 @@ __owur int SSL_SESSION_get_protocol_version(const SSL_SESSION *s);
 __owur const char *SSL_SESSION_get0_hostname(const SSL_SESSION *s);
 __owur int SSL_SESSION_has_ticket(const SSL_SESSION *s);
 __owur unsigned long SSL_SESSION_get_ticket_lifetime_hint(const SSL_SESSION *s);
-void SSL_SESSION_get0_ticket(const SSL_SESSION *s, unsigned char **tick,
+void SSL_SESSION_get0_ticket(const SSL_SESSION *s, const unsigned char **tick,
```

Comment:
It would be really nice to do a pass over the newly added getters that fetch an unsigned char buf, and unify them to all have this form, and make this the future API style. I.e. replace any

```
size_t FOO_get0_bar(const FOO *foo, const unsigned char **bar);
```

and

```
unsigned char *FOO_get0_bar(const FOO *foo, size_t *bar_length);
```

with this form.